README.markdown

```diff
@@ -167,9 +167,8 @@ In addition, there are several known issues with Windows:
 * On Windows Server 2003, only x86 is supported, and the `arch` parameter is ignored. If you try to force an upgrade to x64, Puppet installs the x86 version with no error message.
 * On Windows Server 2003 with Puppet Enterprise, the default download location is unreachable. You can work around this issue by specifying an alternate download URL in the `source` parameter.
  
-Specifically in the 1.2.0 Release:
+Specifically in the 1.3.0 Release:
```

Comment:
Ah, I wasn't sure when that bit should be updated. Just pushed an update that leaves that alone for now.